backend_new/src/controllers/application.controller.ts

```diff
@@ -45,4 +63,61 @@ export class ApplicationController {
   async retrieve(@Param('applicationId') applicationId: string) {
     return this.applicationService.findOne(applicationId);
   }
+
+  @Post()
+  @ApiOperation({ summary: 'Create application', operationId: 'create' })
+  @ApiOkResponse({ type: Application })
+  async create(@Body() dto: ApplicationCreate): Promise<Application> {
+    return await this.applicationService.create(dto, false);
+  }
+
+  @Post(`submit`)
```

Comment:
Now that the applications and applications-submission controllers are combined I think it would be useful to update the "summary" for these two so it is easier to identify the use case to use each of them (applicant submitting vs a partner entering a new application).

Reply:
I made some changes to the summaries lemme know what you think